spec/models/bet_spec.rb

```diff
@@ -0,0 +1,114 @@
+require 'rails_helper'
+
+RSpec.describe Bet, type: :model do
+  let(:bob)   { User.create(username: 'Bob')   }
+  let(:peter) { User.create(username: 'Peter') }
+  let(:jack)  { User.create(username: 'Jack')  }
+
+  let(:bet1) do
+    Bet.create(bet: 'Superbowl: Atlanta vs New England, place your bets!')
+  end
+
+  let(:option1) { BetOption.create(option_text: 'Atlanta wins') }
+  let(:option2) { BetOption.create(option_text: 'New England wins') }
+
+  let(:ub1) do
+    UserBet.create(user:       bob,
+                   bet:        bet1,
+                   bet_option: option1,
+                   amount_bet: 100)
+  end
+
+  let(:ub2) do
+    UserBet.create(user:       peter,
+                   bet:        bet1,
+                   bet_option: option1,
+                   amount_bet: 50)
+  end
+
+  let(:ub3) do
+    UserBet.create(user:       jack,
+                   bet:        bet1,
+                   bet_option: option2,
+                   amount_bet: 300)
+  end
+
+  let(:ub) do
+    ub1
+    ub2
+    ub3
+  end
+
+  context_string = 'Bet: Superbowl - Atlanta vs New England;\n' \
```

Comment:
Ive changed the rubocop so that specs wont care if your lines are long. 

In specs its more important to write them so that they are readable and helpful, so writing long context statements is allows. 

This is one of the few cases where saving this string to a variable is bad because it hides the content which should be very obvious for future devs reading this file.